integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/MonitoringUtils.java

```diff
@@ -104,6 +111,31 @@ public static void buildMonitoringExporterApp(String monitoringExporterSrcDir, S
         Files.copy(srcFile, targetFile, StandardCopyOption.REPLACE_EXISTING));
   }
 
+  /**
+   * Clone monitoring exporter github src.
+   * @param monitoringExporterSrcDir directory containing github monitoring exporter
+   */
+  public static void cloneMonitoringExporter(String monitoringExporterSrcDir) {
+    LoggingFacade logger = getLogger();
+    logger.info("create a staging location for monitoring exporter github");
+    Path monitoringTemp = Paths.get(monitoringExporterSrcDir);
+    assertDoesNotThrow(() -> org.apache.commons.io.FileUtils.deleteDirectory(monitoringTemp.toFile()));
```

Comment:
you can static import org.apache.commons.io.FileUtils.deleteDirectory

Reply:
fixed